packages/ui/src/components/toast/toaster.tsx

```diff
@@ -0,0 +1,41 @@
+import { Icon, Layout, Toast, useToast } from '@/components'
+import { cn } from '@/utils/cn'
+
+export function Toaster() {
+  const { toasts } = useToast()
+
+  return (
+    <>
+      {toasts.map(({ id, title, description, action, variant, showIcon = true, ...props }) => (
+        <Toast.Root key={id} variant={variant} {...props}>
+          <Layout.Horizontal gap="space-x-2.5" className="items-center">
+            {showIcon && (
+              <>
+                {variant === 'success' && <Icon name="checkbox" className="text-icons-success" size={12} />}
+                {variant === 'destructive' && (
+                  <Icon name="warning-triangle-outline" size={12} className="text-toast-foreground-danger" />
+                )}
+                {variant === 'failed' && <Icon name="cross" className="text-icons-danger" size={12} />}
+              </>
+            )}
+            <Layout.Horizontal gap="space-x-1">
```

Comment:
The title and description running into each other seems a little strange. What if the title and description aren't very short? They _happen_ to be short in the case of creating a repo, but across Harness we have dozens of usages of toasts and some have full sentences. e.g. how would this look if we had a title of "Unable to create feature flag" and a description of "We were unable to create your feature flag due to an OPA policy violation. Please check and try again"?

Reply:
even repo names can be very long. This format also makes it look grammatically incorrect.
For eg. `Import in progress canary-4` is wrong.. maybe it should be `Import in progress for repo "canary-4"`